tests/httpwrappers/tests.py

```diff
@@ -662,6 +662,11 @@ def test_streaming_response(self):
 
         r = StreamingHttpResponse(iter(['hello', 'world']))
         self.assertEqual(r.getvalue(), b'helloworld')
+    
+    def test_streaming_response_repr(self):
+        r = StreamingHttpResponse(iter(['hello', 'café']))
+        expected_repr_response = '<StreamingHttpResponse status_code=200>'
+        self.assertEqual(r.__repr__(), expected_repr_response)
```

Comment:
There is not need for an extra variable (`expected_repr_response`), also, we should call `__repr__` directly:
```suggestion
        self.assertEqual(repr(r), '<StreamingHttpResponse status_code=200>')
```

Reply:
Can you add a similar test for the `FileResponse`?

Reply:
sure!